snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -444,33 +444,36 @@
         }
     }
 
-    // Listen for clicks on links that point to 'about:accounts' and
-    // convert them showFirefoxAccounts Events.
-    document.addEventListener('click', function(event) {
+    // Listen for clicks on links, send metrics and handle
+    // about:account custom links.
+    var topSection = document.getElementById('topSection');
+    topSection.addEventListener('click', function(event) {
         var target = event.target;
-        while (target && target.tagName && target.tagName.toLowerCase() !== 'a') {
+        while (target.tagName && target.tagName.toLowerCase() !== 'a') {
+            // Do not track clicks outside topSection.
+            if (target.id === 'topSection') {
```

Comment:
Seeing that we are adding the `eventListener` to the element with `id`, `topSection` above, do we still need this check? There should not be more than one element on the page with an `id` of `topSection` correct?